sdk/identity/azure-identity/azure/identity/_credentials/linux_vscode_adapter.py

```diff
@@ -66,6 +66,15 @@ def _get_refresh_token(service_name, account_name):
     if sys.version_info[0] < 3:
         raise NotImplementedError("Not supported on Python 2.7")
 
+    try:
+        import platform
+        distro = platform.uname()
+        if sys.version_info[0] == 3 and sys.version_info[1] == 8 \
+                and not (distro[1] == "redhat" or distro[1] == "ubuntu"):
+            raise NotImplementedError("Not supported")
+    except Exception:  # pylint: disable=broad-except
```

Comment:
If you're anticipating errors from indexing `distro` here, `"redhat" in distro or "ubuntu" in distro` might be better.